src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int colorInt = new Random().nextInt(Colors.values().length);
+        return Colors.values()[colorInt].toString();
```

Comment:
1) A new instance of random will be created on each invocation of getRandomColor() method, which is bad. It's better to store it at the class level
2) Colors.values() expression used twice here, it's better to store it on the class level too for further reuse
https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#think-about-which-variables-should-be-local-in-the-method-and-which-should-be-class-level